docs/read_and_plot_ismn_data/Read and plot ISMN data.rst

```diff
@@ -650,4 +650,43 @@ network and station to plot data from
     </div>
 
 
+Selection of ISMN stations by landcover or climate classification:
```

Comment:
I see  landcover for 2010 is included elsewhere. why not also include it here?

Reply:
included 2010